features/manage-position/YourLiquidations.tsx

```diff
@@ -56,7 +56,7 @@ const YourLiquidations = () => {
     liquidations.map((liq: LiquidationState) => {
       let maxDisputablePrice =
         liq.tokensOutstanding > 0 && collReqFromWei > 0
-          ? liq.liquidatedCollateral / (liq.tokensOutstanding / collReqFromWei)
+          ? liq.liquidatedCollateral / (liq.tokensOutstanding * collReqFromWei)
```

Comment:
Outside scope of this PR but this was an easy fix to a typo from my latest YourLiquidations PR.